Running make proto has some errors

While running `make proto`, I am facing some issues, here is a traceback - 
```
installing gogofast
generating code
~/golib/src/github.com/yashrsharma44/go-grpc-middleware/grpctesting/testpb ~/golib/src/github.com/yashrsharma44/go-grpc-middleware
~/golib/src/github.com/yashrsharma44/go-grpc-middleware
~/golib/src/github.com/yashrsharma44/go-grpc-middleware/grpctesting/gogotestpb ~/golib/src/github.com/yashrsharma44/go-grpc-middleware
github.com/gogo/protobuf/gogoproto/gogo.proto: File not found.
fields.proto: Import "github.com/gogo/protobuf/gogoproto/gogo.proto" was not found or had errors.
Makefile:28: recipe for target 'proto' failed
make: *** [proto] Error 1
```

I looked into other resources and found that @johanbrandhorst has a blog on the issue - https://jbrandhorst.com/post/gogoproto/
But there wasn't any workaround for the above, hence I decided to post this issue and ask the maintainers :stuck_out_tongue:

--- a/grpctesting/gogotestpb/fields.proto
+++ b/grpctesting/gogotestpb/fields.proto
@@ -3,7 +3,7 @@ syntax = "proto3";
 //  This file is used for grpctesting discovery of log fields from requests using reflection and gogo proto more tags.
 package grpc_middleware.gogotestpb;
 
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "gogoproto/gogo.proto";
 import "google/protobuf/timestamp.proto";
 
 option (gogoproto.gogoproto_import) = false;

--- a/scripts/genproto.sh
+++ b/scripts/genproto.sh
@@ -13,11 +13,6 @@ if ! [[ "$0" =~ "scripts/genproto.sh" ]]; then
 	exit 255
 fi
 
-if ! [[ $(${PROTOC_BIN} --version) =~ "3.4.0" ]]; then
-	echo "could not find protoc 3.4.0, is it installed + in PATH?"
-	exit 255
-fi
-
 echo "installing gogofast"
 GO111MODULE=on go install "github.com/gogo/protobuf/protoc-gen-gogofast"
 

